automated install/basic-install.sh

```diff
@@ -1434,99 +1995,139 @@ main() {
     # Clone/Update the repos
     clone_or_update_repos
 
-       # Install packages used by the Pi-hole
-    if [[ ${INSTALL_WEB} == true ]]; then
+    # Install packages used by the Pi-hole
+    if [[ "${INSTALL_WEB}" == true ]]; then
+      # Install the Web dependencies
       DEPS=("${PIHOLE_DEPS[@]}" "${PIHOLE_WEB_DEPS[@]}")
+    # Otherwise,
     else
+      # just install the Core dependencies
       DEPS=("${PIHOLE_DEPS[@]}")
     fi
     install_dependent_packages DEPS[@]
 
+    if [[ -x "$(command -v systemctl)" ]]; then
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(systemctl is-enabled lighttpd | grep -c 'enabled' || true)
+    else
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(service lighttpd status | awk '/Loaded:/ {print $0}' | grep -c 'enabled' || true)
+    fi
 
     # Install and log everything to a file
     installPihole | tee ${tmpLog}
   else
     # Clone/Update the repos
     clone_or_update_repos
 
-    # Source ${setupVars} for use in the rest of the functions.
+    # Source ${setupVars} for use in the rest of the functions
     source ${setupVars}
 
     # Install packages used by the Pi-hole
-    if [[ ${INSTALL_WEB} == true ]]; then
+    if [[ "${INSTALL_WEB}" == true ]]; then
+      # Install the Web dependencies
       DEPS=("${PIHOLE_DEPS[@]}" "${PIHOLE_WEB_DEPS[@]}")
+    # Otherwise,
     else
+      # just install the Core dependencies
       DEPS=("${PIHOLE_DEPS[@]}")
     fi
     install_dependent_packages DEPS[@]
 
+    if [[ -x "$(command -v systemctl)" ]]; then
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(systemctl is-enabled lighttpd | grep -c 'enabled' || true)
+    else
+      # Value will either be 1, if true, or 0
+      LIGHTTPD_ENABLED=$(service lighttpd status | awk '/Loaded:/ {print $0}' | grep -c 'enabled' || true)
+    fi
+
```

Comment:
Lines 2037 - 2043 is duplicate of 2009 - 2015.

Reply:
Duplicate on purpose as it's in two different branches of an it's in the `else` branch of `if [[ "${userUpdateVars}" == false ]]`.